Source/Configuration/ECShared.xcconfig

```diff
@@ -193,12 +193,40 @@ ASSETCATALOG_WARNINGS = YES
 
 IBSC_WARNINGS = YES
 
+// XC 10.2 Additions
```

Comment:
`ECShared.xcconfig` does not import our global shared xcconfig. Thus setting the recommended warnings here.